randomkick.py

```diff
@@ -135,7 +135,7 @@ async def kick_user(delay, *, warn):
                 await asyncio.sleep(random.randint(7, 10))
             await bot.send_message(GROUP, 'Oh darn! That was close 😅')
 
-        warn_message = await bot.get_message(GROUP, event.id) if warn else None
+        warn_message = await bot.get_messages(GROUP, event.id) if warn else None
```

Comment:
The second parameter is limit, not the ID to fetch. This will attempt to get the entire history.